resources/lib/fanarttv.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
```

Comment:
Shebang is meant for executable scripts in *nix environment, not for importable modules.